06-files-input-and-output/high_scores.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import csv
+import os
+
+def parse_csv(csv_file):
```

Comment:
Use `csv_path` instead. You already use `csv_file` in the function's body. (In my initial comment I wrote csv_file but then edited it to csv_path. You can see the comments after edits in gitHub rather than in your inbox).